tests/epyccel/test_arrays.py

```diff
@@ -1203,9 +1203,7 @@ def test_array_real_2d_C_array_initialization(language):
     assert np.array_equal(x1, x2)
 
 @pytest.mark.parametrize( 'language', [
-        pytest.param("c", marks = [
-            pytest.mark.skip(reason="array function doesn't handle list of variables. See #752"),
-            pytest.mark.c]),
+        pytest.param("c", marks = pytest.mark.c),
```

Comment:
Does this test run with `language=python`? If yes then you can remove the parametrization. If no then do you know why not?

Same comment below

Reply:
Yes I will check these ones

Reply:
@ohachim